www/pages/stop/stop.html

```diff
@@ -3,7 +3,7 @@
   <ion-nav-buttons side="secondary">
     <button id="like" data-tap-disabled="true" ng-class="{'button button-icon icon ion-ios-heart': liked === true, 'button button-icon icon ion-ios-heart-outline': liked === false}"
       ng-click="liked= !liked; toggleHeart(liked)"
-      aria-label="Like">
+      aria-label="{{liked ? 'Remove from liked' : 'Add to liked'}}">
```

Comment:
I was going back and forth on what to call it, because at first "Like" and "Unlike" seemed too vague and confusing. Now I'm thinking 'Like stop' and 'Unlike stop' would be the best format.